working-groups/meta/README.md

```diff
@@ -0,0 +1,31 @@
+# Meta Working Group
+![working group status: active][status]
+
+This working group is dedicated to fleshout details of how the compiler team will organize itself.
+
+- **Leads:** [@nikomatsakis][nikomatsakis]
+- **Meeting Notes:** [All](NOTES.md)
+
+[status]: https://img.shields.io/badge/status-active-brightgreen.svg?style=for-the-badge
+
+## What is the goal of this working group?
+This working group aims to accomplish the following:
+
+- Organizing working groups
+- Organizing rustc calendar
+- Organizing videos and lectures
+- Organizing communication between members
```

Comment:
If they are reworded slightly, this might actually be a good spot to put the "big list of ideas" that the working group wants to think about and consider (from the meeting paper document).

Reply:
I wonder if we should create a ROADMAP.md and try to put information there -- or else link to a dropbox paper. One or the other really.

Reply:
Yeah, a ROADMAP.md file looks good.

Reply:
Done.